.github/workflows/validate-all.yml

```diff
@@ -16,8 +16,28 @@ on:
       - cron: '0 0 * * *'
 
 jobs:
+  perl_validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup conda
+        uses: s-weigand/setup-conda@v1
+        with:
+          update-conda: true
+          conda-channels: anaconda, bioconda, conda-forge
+      - name: Install perl-atlas-modules
+        run: conda create -n perl-atlas -c ebi-gene-expression-group perl-atlas-modules
+      - name: Conda init bash
+        run: conda init bash
```

Comment:
We might be able to remove this, since the activate is being forced in the next step (not with conda activate ...)

Reply:
Feel free to remove it!!! @pcm32